examples/control_character/run.py

```diff
@@ -0,0 +1,193 @@
+        get_num_bones_func = instance.unreal_service.find_function_by_name(uclass=skeletal_mesh_component_uclass, function_name="GetNumBones")
+        get_bone_name_func = instance.unreal_service.find_function_by_name(uclass=skeletal_mesh_component_uclass, function_name="GetBoneName")
+        get_bone_transform_func = instance.unreal_service.find_function_by_name(uclass=skeletal_mesh_component_uclass, function_name="GetBoneTransform")
+        create_input_component_func = instance.unreal_service.find_function_by_name(uclass=actor_uclass, function_name="CreateInputComponent")
+
+        gameplay_statics_default_object = instance.unreal_service.get_default_object(uclass=gameplay_statics_uclass, create_if_needed=False)
+        manny_simple_uobject = instance.unreal_service.load_object(class_name="UObject", outer=0, name="/Game/Characters/Mannequins/Meshes/SKM_Manny_Simple.SKM_Manny_Simple")
+
+        bp_third_person_class = instance.unreal_service.load_object(
+            class_name="UObject", outer=0, name="/Game/ThirdPerson/Blueprints/BP_ThirdPersonCharacter.BP_ThirdPersonCharacter_C")
+        bp_third_person_actor1 = instance.unreal_service.spawn_actor_from_uclass(
+            uclass=bp_third_person_class,
+            location={"X": 0.0, "Y": 100.0, "Z": 150.0},
+            rotation={"Roll": 0.0, "Pitch": 0.0, "Yaw": 0.0},
+            spawn_parameters={"Name": "third_person_actor1", "SpawnCollisionHandlingOverride": "AlwaysSpawn"}
+        )
+        bp_third_person_actor2 = instance.unreal_service.spawn_actor_from_uclass(
+            uclass=bp_third_person_class,
+            location={"X": 0.0, "Y": 200.0, "Z": 150.0},
+            rotation={"Roll": 0.0, "Pitch": 0.0, "Yaw": 0.0},
+            spawn_parameters={"Name": "third_person_actor2", "SpawnCollisionHandlingOverride": "AlwaysSpawn"}
+        )
+
+        # set bRunPhysicsWithNoController to True to control pawn without controller
+        character_movement_component1 = instance.unreal_service.get_component_by_class(actor=bp_third_person_actor1, uclass=character_movement_component_uclass)
+        character_movement_component2 = instance.unreal_service.get_component_by_class(actor=bp_third_person_actor2, uclass=character_movement_component_uclass)
+
+        instance.unreal_service.set_object_properties_for_uobject(uobject=character_movement_component1, properties={"bRunPhysicsWithNoController": True})
+        instance.unreal_service.set_object_properties_for_uobject(uobject=character_movement_component2, properties={"bRunPhysicsWithNoController": True})
+        instance.unreal_service.call_function(uobject=character_movement_component1, ufunction=set_movement_mode_func, args={"NewMovementMode": "MOVE_Walking"})
+        instance.unreal_service.call_function(uobject=character_movement_component2, ufunction=set_movement_mode_func, args={"NewMovementMode": "MOVE_Walking"})
+
+        # initialize input component
```

Comment:
I'm fine with using the `by_class` variants here, but please register the types in `UnrealClassRegistrar`.